src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -28,62 +29,131 @@ internal abstract partial class AbstractInitializeParameterCodeRefactoringProvid
         protected abstract SyntaxNode GetBody(SyntaxNode functionDeclaration);
         protected abstract SyntaxNode GetTypeBlock(SyntaxNode node);
 
+        protected virtual ImmutableArray<SyntaxNode> GetParameters(SyntaxNode parameterNode)
+        {
+            return default;
```

Comment:
why?  won't everything need to override this?  shouldn't this method be asbtract?